### ISSUE TITLE
Setting unknown property: barrelstrength\sproutseo\models\Metadata::section

### ISSUE THREAD

**Issue by engram-design** [barrelstrength/craft-sprout-seo#167]
Upgrading a Craft 2 site, with template code:
```
{% do craft.sproutSeo.meta({
  section: 'cart'
}) %}
```

But getting the error as attached. Looking at the [docs](https://sprout.barrelstrengthdesign.com/docs/seo/template-overrides.html#templating) `section` is still a valid attribute right?

**Comment by engram-design** [comment]
Any follow up on this? Is there a change in behaviour and the docs need updating, or am I doing something wrong?

**Comment by BenParizek** [comment]
@engram-design Sections were removed in Sprout SEO for Craft 3  and it appears I need to update the docs to reflect that difference.

The recommended approach now is to set up a Single with a Metadata field with whatever you had your Section doing in the past.

**Comment by engram-design** [comment]
Gotcha, as you can tell I'm not up to date with the changes. I think its rather worrisome that the content that used to be in sections is now completely gone from the database?

**Comment by BenParizek** [comment]
@engram-design Yes. I believe I had a task to add better upgrade documentation that got lost in the shuffle. I'll be sure to update the Craft 3 upgrade notes before I close out this ticket.

We did wait to introduce this for the C2->C3 update as it's a major release and is more likely to be closely reviewed for breaking changes. 

The sections it will matter for most will likely be Singles. One reason we removed the feature is that in the majority of use cases, it was just a redundant and unused layer on top of Channels and other URL-Enabled Elements that were using the Metadata field on the individual entry field layouts.
